### PR TITLE
small translation

### PR DESCRIPTION
**Pull request by teolemon**
small translation

### CODE REVIEW

File: taxonomies/labels.txt [hunk 1]
```diff
@@ -474,6 +474,7 @@ ru:EAC
 wikidata:en:Q18407253
 
 fr:Entrepreneurs + Engagés, E+, Entrepreneurs Engagés
+de:Entrepreneurs + Engagés
```
Comment (stephanegigandet) [inline taxonomies/labels.txt:477]: Why add this?

Comment (teolemon) [reply taxonomies/labels.txt:477]: Some values are stored in German, for some reason 🤷

Comment (hangy) [reply taxonomies/labels.txt:477]: I think that's the usual disruptive apps, probably some older changes, but we shouldn't update the taxonomy to reflect that IMHO. Not having a taxonomy entry for this actually makes it easier to fix the values.

Comment (teolemon) [reply taxonomies/labels.txt:477]: you are perfectly right, I closed the PR

**Pull request closed without merging**